killing-floor-2config.json

```diff
@@ -2,53 +2,14 @@
     {
         "DisplayName": "Starting Map",
```

Comment:
Rather than remove all the options, I'd recommend you take a look at Empyrion for how the Scenario and Custom Scenario inputs work. It should apply here as well.
https://github.com/CubeCoders/AMPTemplates/blob/main/empyrion-galactic-survivalconfig.json

Reply:
I will check it.